openreview/conference/invitation.py

```diff
@@ -1699,13 +1699,12 @@ def post_invitation(note):
         return invitations
 
     def set_revise_submission_invitation(self, conference, notes, content):
-
-        invitations = []
         self.client.post_invitation(SubmissionRevisionInvitation(conference, content))
-        for note in tqdm(notes, total=len(notes), desc='set_revise_submission_invitation'):
-            invitations.append(self.client.post_invitation(PaperSubmissionRevisionInvitation(conference, note, content)))
-
-        return invitations
+        return tools.concurrent_requests(
```

Comment:
most of the methods here are looping the notes to create invitations, do you think we should use concurrent requests for all of them?

Reply:
I can merge this PR and we start changing the other methods little by little

Reply:
I feel a little bit scared about using concurrent requests in python, most of the time works well for me but sometimes one request it gets stuck ... but I will give it a try.

I will merge this and see how it goes.

Reply:
I didn't know this was happening. This may be related to a server issue. Perhaps related to what Zach has reported as well.